### DISCUSSION TITLE
ERR_SSL_PROTOCOL_ERROR

### DISCUSSION THREAD

**Discussion by Gabrycode17** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#809] [Q&A]
when I go to google chrome to get my token to start using the assistant chrome says that "this site can't provide a secure connection" and the error message above and I can't get my code
what can I do??

**Comment by Carlos-Simim** [comment]
Change the address beginning to "http://" instead of "https://"

**Reply by earldanielph** [reply]
this works for me. thanks!

**Reply by ethereum2O** [reply]
damn, I'm looking for this so long, finally thank you so much!

**Comment by Melvin-Abraham** [comment]
This issue has been addressed and an FAQ section for the same has been added. Refer [FAQ: Why am I getting SSL Protocol Error when assistant opens localhost?](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/wiki/Frequently-Asked-Questions-(FAQ)#why-am-i-getting-ssl-protocol-error-when-assistant-opens-localhost)
